srcpkgs/python3-jupyter_console/patches/setuptools.diff

```diff
@@ -1,11 +0,0 @@
---- setup.py
-+++ setup.py
-@@ -51,7 +51,7 @@
- import os
- from glob import glob
- 
--from distutils.core import setup
-+from setuptools import setup
```

Comment:
Could you please add an explanation to this commit that `jupyter_console` now unconditionally uses setuptools rather than distutils (explaining why patch is deleted).

Reply:
I've forked this commit from @ahesford , so I'm not sure about their reason. However, it seems to build fine without. What was the reason for adding patch?

Reply:
Yep. It should be fine without now since recent versions of jupyter_console made the change to `setuptools`. This patch was necessary because `distutils` doesn't generate the `/usr/bin/jupyter-console` script during build.

Reply:
Good. Should I add a comment to the commit message?

Reply:
No. The patch is obsolete and the reason for its removal is readily discerned. If we took space to explain the removal of every obsolete patch, the commit logs would be awash in noise.